src/Shared/Helpers/CompletionCommandArgs.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS_WINUI
```

Comment:
since WINDOWS_WINUI is limited to windows10, let's use WINUI instead

Reply:
The android and iOS condition already include the WinUI targets

Reply:
I understand that android and ios are already included, but you include only windows10, it should not be specific to a version of windows, the same way that we would not put __Android13__ instead of __ANDROID___

Reply:
fixing my other comment about Windows10 will fix the issue here